CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `fill` prop to `PageBlock` and `Box` components.
```

Comment:
It could be a prop named `displayMode` or something and accept the values `['fill', 'default']` and then you could add some other options that just add `w-100` or just add `h-100`

Reply:
It can be done this way. I don't see much value in making the `w-100` and `h-100` separately because the `w-100` is already the default behavior (I just added in this PR to be explicit but it can be removed). Anyway, `displayMode` feels weird. Do you have a better idea?

Reply:
`stretchMode` I think, but the idea is to create a prop that is an enum instead of creating a boolean

Reply:
@matheusps @emersonlaurentino any suggestions?

Reply:
I was just looking at a name for the prop. The idea is precisely to reduce the use of boolean props.

Reply:
I liked the suggestion of @Klynger

Reply:
Which one? @emersonlaurentino

Reply:
`displayMode`

Reply:
but did we get a more specific name?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
This fit may not be the one because only makes sense to us the `fill` and `cover` options, I think...

Reply:
But if we agree in a good name for this prop, I can make the change with no problem

Reply:
in object fit there are these two options. But I don't like that name...

Reply:
Maybe `fit: 'fill' | 'fill-horizontal' | 'fill-vertical' | 'none'`? Or something com fit, like `fitMode`

Reply:
Okay... I'll do it.